### PR TITLE
created login page & logo header component

### PR DESCRIPTION
**Pull request by EvanGrillo**
### Describe The Problem Being Solved:

Creating a LoginPage component/wrapper + a reusable logo header component 

## Designer Request
https://drive.google.com/drive/folders/1WYcHQ3_eRM3SW4avvgoe7cukb_Itt9jl

## LoginPage with LogoHeader imported 

screen shot 2019-01-14 at 7 43 59 pm

### Checklist For Submitter

* [ x] I have documented all new functionality
* [ x] I have screenshotted new UI changes and attached them to this PR

### Checklist For Reviewer
* [ ] Code formatting/static analysis: is the code formatted properly? Does a linter/other form of static analysis reveal any issues? Does the code have appropriately low cyclomatic complexity?
* [ ] Code architecture: concerns are separated, code follows single-responsibility principle, code utilizes OOP, DRY code.
* [ ] Coding best practices: code avoids hard-coded variables, inefficient computations, reinventing the wheel when using frameworks. Code is well-commented and generally readable.
* [ ] Non-functional requirements: Code has accompanying tests (if using TDD). Programmer can easily explain decisions to reviewers.

### CODE REVIEW

File: frontend/src/components/LoginPage/LoginPage.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.pageHeaderDiv {
+	width: 100%;
+	height: 25vw;
```
[the remark this answers is not in the document]
Comment (EvanGrillo) [reply frontend/src/components/LoginPage/LoginPage.css]: never noticed that before today. I'll revisit this

File: frontend/src/components/LogoHeader/LogoHeader.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  handleInputChange(event) {
+      this.setState({ [event.target.name]: event.target.value });
+      console.log("handle change", event.target);
+  }
+
+  render() {
+      return (
+        <div className="wrapper">
+          <div className="pageLogoDiv">
+            <img className="logo" src="https://www.rescuingleftovercuisine.org/assets/RLC_LOGO-900fcd5d7d719b63e43d29ac23775f74f17100194e2690e17833a091b016da85.png" />
```
[the remark this answers is not in the document]
Comment (EvanGrillo) [reply frontend/src/components/LogoHeader/LogoHeader.js]: this is temporary until we have the asset + directory to store it set up

[the remark this answers is not in the document]
Comment (EvanGrillo) [reply frontend/src/components/LogoHeader/LogoHeader.js]: Also, will we store branding assets in the project or somewhere like Amazon with other files?

File: frontend/src/components/LogoHeader/LogoHeader.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import "./LogoHeader.css"
+
+class LogoHeader extends Component {
+
+  handleInputChange(event) {
+      this.setState({ [event.target.name]: event.target.value });
+      console.log("handle change", event.target);
+  }
```
[the remark this answers is not in the document]
Comment (EvanGrillo) [reply frontend/src/components/LogoHeader/LogoHeader.js]: standard react from the documentation. to simplify - without it your input value won't change the state of the div (e.g. if you typing into a form input, it won't capture the value)

[the remark this answers is not in the document]
Comment (vhoof) [reply frontend/src/components/LogoHeader/LogoHeader.js]: Right. I suppose I should've asked if we need it in this component?

[the remark this answers is not in the document]
Comment (EvanGrillo) [reply frontend/src/components/LogoHeader/LogoHeader.js]: No thanks for pointing that out. We're importing the login/signup components

File: frontend/src/components/LogoHeader/LogoHeader.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (vhoof) [reply frontend/src/components/LogoHeader/LogoHeader.js]: Hmm. Do we not have the logo in `frontend/src/assets`?

I don't expect we'll store them in Amazon for now. Maybe down the road.

[the remark this answers is not in the document]
Comment (EvanGrillo) [reply frontend/src/components/LogoHeader/LogoHeader.js]: We do - I made the correction in the latest commit. A bit trigger happy over here today. Should be a bit better off with the last one

**Comment by vhoof** [comment]
I finally found some time to actually take a look at the rendering. Sorry for the delay on that.

As Maryann mentioned, t looks like some CSS is interfering with the home page. I'm not sure if your branch is up-to-date with master, but at present, master renders OK. In this branch, the home page looks like:
screen shot 2019-01-24 at 5 01 44 pm

The top of the page is cut off, and things are a bit jumbled. Not sure where the problem is there, though I suspect that something is overriding the styles there.

On the login page, it looks pretty good. It seems like the boxes containing the forms should have significantly more padding so that they fill the page more. Right now they look a little short and fat. It may help to make the form elements a little more narrow.

Additionally, it seems like the distance between them should be smaller, growing with the size of the page only up to a certain point (like x% of the width up to 100px or something). It'd be better to grow the white space on either side of them if necessary.

The text appears to be black, but in the spec is actually a sort of grey (which appears to be used throughout the site). I'm not sure what exactly that color is, but I assume we have that information somewhere (or can use a color picker).

The form inputs need to lose their "3D" shading along the borders and should also be grey. And it'd be great if the first/last name fields could together equal the length of the email field (so each line is the same length).

The font sizes also look a bit off, with the headers seemingly a little small and the "Forgot my password" link a bit large.

As a reminder, here's the design:
login_page

**Comment by vhoof** [comment]
Woops, accidentally hit the merge button. I need to set up some more validation on that.

**Comment by vhoof** [comment]
@EvanGrillo I reverted the PR and such. Unfortunately that means you'll need to re-open it. My apologies!

**Comment by vhoof** [comment]
@EvanGrillo Sorry for the confusing messages. I thought I'd accidentally merged this. It looks like it still needs some work, so I reverted it, which means you'll need to reopen it. There are some things in the above comment that need to be addressed, but it's coming together!

**Pull request merged**